addons/im_livechat/static/tests/embed/livechat_button.test.js

```diff
@@ -30,6 +30,7 @@ test("open/close temporary channel", async () => {
     await contains(".o-livechat-LivechatButton", { count: 0 });
     expect(Boolean(env.services["im_livechat.livechat"].thread)).toBe(true);
     await click("[title*='Close Chat Window']");
+    await click("button[title='Leave conversation']");
```

Comment:
No "leave confirmation" on temporary channel, there's no effective conversation, so there's no risk in leaving it.